src/ILVerification/tests/ILTests/AccessTests.ilproj

```diff
@@ -0,0 +1,3 @@
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="ILTests.targets" />
```

Comment:
It would be possible to avoid these boilerplate .ilproj files by writing msbuild traversal rule. I have opted for the boilerplate files since there is not that many of them and they are much easier to understand than msbuild traversal rules.